### ISSUE TITLE
Win WebView 2 Scroll Issue

### ISSUE THREAD

**Issue by Dxg125** [react-native-webview/react-native-webview#3102]
**Bug description:**

TL;DR After clicking / focusing (via mouse) inside the WebView2 (windows only), the Mouse-Wheel is getting ignored.

On Opening (without clicking, and autofocus from windows/system) scrolling is possible until the first mouse-click.
Dragging the scrollbar or scrolling with the middle-click (the middle mouse button) does still work.
The middle-click icon / pointer is missing tho, the default pointer gets shown instead.

**To Reproduce:**

See / Download Example App.tsx: https://github.com/Dxg125/React-Native-WebView2-Scroll-Issue

Basically
Install React Native & React Native WebView
Install React Native Windows

Run App, click inside of container, mouse-scroll will not be working any longer.
But if you (after compilation) open the app, (from windows task bar) you can scroll until the first click.

I've tried something like setInterval(() => WebView.requestFocus(), 1); -> didnt work as workaround.

I've also tried it with the initial WebView2 Engine, and the newest one (ref bug #3093).

**Expected behavior:**

The Scroll should work as intended, or some method as workaround for re-focussing the window.

**Edit:**
This may be related to: https://github.com/Microsoft/react-native-windows/issues/1006

**Environment:**
 - OS: Windows 10 Enterprise
 - OS version: 22H2 / 19045.3324
 - react-native version: 0.72.4
 - react-native-webview version: 13.3.1
 - react-native-windows version: 0.72.6

**Comment by chiaramooney** [comment]
A few things to check...In the getting started instructions for Windows (https://microsoft.github.io/react-native-windows/docs/customizing-sdk-versions) there is a note about "enable scroll with Touch for the WebView component" and the workaround needed for this bug. It may be possible this bug has other side effects besides just touch scrolling. I would make sure your app has the workaround enabled. 

Next, it may be that this bug comes from the WebView2 control itself. I would check more recent versions of WebView2 (by updating your app to use WinUI2.8+) to see if the bug resolves itself. Otherwise we can do some testing on our end to see if this reproduces in a plain WinUI app.

**Comment by Dxg125** [comment]
Hey, first off thank you for your answer.

Im maybe blind but i cannot find this "note", not in the link provided, nor by a google search.
If you're talking about the Configurable Params, yes i've set them to enableScroll (nestedScrollEnabled too). 
Still did not work tho. I also could never get the onScroll Event to fire.

I've tried with WinUI 2.8.5 (the latest i believe, and also with the latest Webview).

After ditching and fiddling around with this, i had to switch projects as i couldn't get it to work at all.
So i guess, if nobody else has this weird behaviour, you can close this bug imo.

**Comment by chiaramooney** [comment]
Apologies wrong link was copied. Here it is: https://github.com/react-native-webview/react-native-webview/blob/master/docs/Getting-Started.md#windows. The note links to this issue () that explains the workaround https://microsoft.github.io/react-native-windows/docs/ReactRootView#isperspectiveenabled. For an example of the workaround used in an app see here: https://github.com/microsoft/react-native-gallery/blob/536ed0a4904573b0b64521cbb31e4955528d4605/windows/rngallery/MainPage.cpp#L21.

**Comment by Dxg125** [comment]
Hey, sorry for the late answer, got it tested.
The Behaviour is the same: as the window is getting clicked, the mouse scroll will stop working.

**Comment by chiaramooney** [comment]
Ok thanks for trying the workaround. We will take a look at the issue.
